packages/ui/src/components/va-dropdown/VaDropdown.stories.ts

```diff
@@ -1,12 +1,386 @@
+      <VaValue #default="v" :default-value="true">
+        <button @click="v.value = !v.value">{{ v.value ? 'Short' : 'Long' }}</button>
+        <br/>
+        <va-dropdown placement="bottom-end">
+          <template #anchor>
+            <div>
+              Custom width anchor
+            </div>
+          </template>
+
+          <template #default="{ getAnchorWidth }">
+            <va-dropdown-content :style="{ minWidth: getAnchorWidth() }">
+              {{ v.value ? 'Short' : 'Loooong drooooopdown wwiiiiiidth' }}
+            </va-dropdown-content>
+          </template>
+        </va-dropdown>
+      </VaValue>
+    `,
+})
+
+export const Cursor = () => ({
+  components: { VaDropdown },
+  template: `
+      <va-dropdown
+          trigger="right-click"
+          placement="bottom-start"
+          cursor
+      >
+        <template #anchor>
+          <div style="width: 8rem; height: 8rem; border: var(--border-dashed)">
+            Right click
+          </div>
+        </template>
+        Ctx dropdown
+      </va-dropdown>
+    `,
+})
+
+export const CursorWithTarget = () => ({
+  components: { VaDropdown },
+  setup () {
+    return {
+      cursorTarget: ref(null),
+    }
+  },
+  template: `
+      <va-dropdown
+          trigger="right-click"
+          placement="bottom-start"
+          :target="cursorTarget"
+          cursor
+      >
+        <template #anchor>
+          <div
+              style="width: 8rem; height: 8rem; border: var(--border-dashed)"
+              ref="cursorTarget"
+          >
+            Right click
+          </div>
+        </template>
+        Ctx dropdown
+      </va-dropdown>
+    `,
+})
+
+export const AnchorSelector = () => ({
+  components: { VaDropdown },
+  template: `
+      <div
+          style="border: var(--border-dashed)"
+          id="anchor-selector"
+      >
+        Click me
+      </div>
+      <va-dropdown
+          anchor-selector="#anchor-selector"
+      >
+        Anchor selector
+      </va-dropdown>
+    `,
+})
+
+export const InnerAnchorSelector = () => ({
   components: { VaDropdown },
-  template: '<VaDropdown/>',
+  template: `
+      <va-dropdown
+          inner-anchor-selector="#inner-anchor-selector"
+      >
+        <template #anchor>
+          <div style="border: var(--border-dashed)">
+            <div
+                style="margin: 1rem; border: var(--border-dashed)"
+                id="inner-anchor-selector"
+            >
+              Click me
+            </div>
+          </div>
+        </template>
+        Inner anchor selector
+      </va-dropdown>
+    `,
+})
+
+export const AnchorBoundProps = () => ({
+  components: { VaDropdown, VaIcon },
+  template: `
+      <va-dropdown>
+        <template #anchor="{ isOpened }">
+          <div>
+            <va-icon :name="isOpened ? 'va-arrow-up' : 'va-arrow-down'"/>
+          </div>
+        </template>
+        Clicked
+      </va-dropdown>
+    `,
+})
+
+export const KeepHeight = () => ({
```

Comment:
Really unclear what this example shows (I have no idea 🤷).

If we can showcase that via interaction or naming - we should.